backend/src/services/smtp/smtp.ts

```diff
@@ -37,11 +37,11 @@ export async function requestReviewForRelease(entity: string, review: ReviewDoc,
   }
 
   const emailContent = buildEmail(
-    `Release ${release.semver} has been created for model ${release.modelId}`,
+    `Release ${release.semver} for model ${release.modelId} is ready for your review`,
```

Comment:
Under a previous ticket, BAI-1383, all emails that require a user to do something should now have ""ACTION REQUIRED:" prepended to the subject?